api/api_test.go

```diff
@@ -80,10 +115,32 @@ func newTestServer(t *testing.T) *testServer {
 		t.Fatalf("err: %s", err)
 	}
 
+	return &testServer{
+		pid:        cmd.Process.Pid,
+		dataDir:    dataDir,
+		configFile: configFile.Name(),
+	}
+}
+
+func makeClient(t *testing.T) (*Client, *testServer) {
```

Comment:
We should probably have a `DefaultConfig()`-esque method instead of having a global variable and resetting the address for each test that uses it.